pom.xml

```diff
@@ -123,10 +123,9 @@
 			</dependency>
 
 			<dependency>
-				<groupId>io.rest-assured</groupId>
-				<artifactId>rest-assured</artifactId>
-				<version>${restassured.version}</version>
-				<scope>test</scope>
+				<groupId>org.projectreactor</groupId>
+				<artifactId>reactor-spring</artifactId>
```

Comment:
Where is this used? All the imports seem to be boot or framework specific.

Reply:
I was getting a weird exception from the client when this dependency wasn't present